.github/workflows/fix_permissions.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+datadir=$1
+echo "Attempting to fix ${datadir} permissions to be own by app:app"
+chown -R app:app $datadir
```

Comment:
Why do we need to do this?

No need to reply in PR, just #PleaseUpdateComment (#PUC) in the code..